### ISSUE TITLE
add healthcheck to the docker image

### ISSUE THREAD

**Issue by ToeiRei** [jellyfin/jellyfin-metapackages#4]
It would be cool to actually be able to tell from docker if things are running.

In the dockerfile you'd just need a small extension like that:

`HEALTHCHECK --interval=5m --timeout=3s  CMD curl -f http://localhost/ || exit 1`

That way docker is able to tell if the container is healthy due to the exit code of the command. 

Quick and dirty: add curl
Better solution: actually have something like jellifin --healthcheck implemented

((reference: https://docs.docker.com/engine/reference/builder/#healthcheck))

**Comment by Ornias1993** [comment]
Agreed.
It's also part of the CIS docker security benchmark v1.2.0, which should be taken serieusly imho.

**Comment by chadmiller** [comment]
_Jellyfin exposes the /health endpoint designated for checking the status of the underlying service. Currently this will verify HTTP and database connectivity and return a 200 OK response if successful. You can see this for yourself by using curl:_

_curl -i http://myserver:8096/health_

https://jellyfin.org/docs/general/networking/monitoring.html

**Comment by Ornias1993** [comment]
@chadmiller Thats not what we refered to. We refered to the docker image not the lack of endpoint.

**Comment by chadmiller** [comment]
@Ornias1993 Oh, I see only a test that a web server is running, not anything about a health endpoint.

**Comment by Ornias1993** [comment]
@chadmiller No, the OP asked to have ANY healthcheck INCLUDED in the dockerfile.
As there is none at all.

**Comment by crobibero** [comment]
The images now have a healthcheck configured.
